data/data.encrypted

```diff
@@ -1,100 +0,0 @@
-soDS/wNhmA4MMMLLVyEV0A==
```

Comment:
How will this change the workflow for someone who gets a fresh copy of the repo?

Reply:
They will just need to update config.json with all parameters and run "sh build.sh"